### PR TITLE
docs: add a blog to announce Terrafrom Bytebase Provider

### PR DESCRIPTION
**Pull request by Candybase**
A new PR (https://github.com/bytebase/bytebase.com/pull/756) is committed. The PR is abandoned.

### CODE REVIEW

File: content/blog/introducing-terraform-bytebase-provider.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+## Why Terraform
+
+Terraform is an open-source tool that allows you to manage and provision infrastructure using code. It is commonly used for infrastructure as code (IaC) and is known for its ability to work across multiple cloud providers, reusability, versioning, flexibility, and its active community. Terraform providers are plugins that enable Terraform to communicate and manage resources on various cloud platforms such as AWS, Google Cloud, and others.
+
+Some Bytebase customers from overseas use Terraform to manage their infrastructure resources, and they want to use it to manage their Bytebase resources such as environments, database instances, and policies. Previously, we offered customers Bytebase Console (a web-based GUI tool) and OpenAPI to create, update and archive Bytebase resources. However, we are excited to reveal our latest feature, Terraform Bytebase Provider, which enables you to manage Bytebase resources through Terraform. This is a great opportunity for you to simplify and streamline your infrastructure management.
```
Comment (adela-bytebase) [inline content/blog/introducing-terraform-bytebase-provider.md:15]: no need to mention "from overseas"?

File: content/blog/introducing-terraform-bytebase-provider.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Terraform Bytebase Provider
```
Comment (adela-bytebase) [inline content/blog/introducing-terraform-bytebase-provider.md:16]: title?

File: content/blog/introducing-terraform-bytebase-provider.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+published_at: 2023/01/20 13:37:17
+feature_image: /static/blog/introducing-terraform-bytebase-provider/cover.webp
+tags: Announcement
+featured: true
+description: In this article, we just introduced a new feature, Terraform Bytebase Provider, that provides you a convenient way to manage Bytebase resources, including environments, database instances, roles, and policies. 
+---
+
+## Why Terraform
+
+Terraform is an open-source tool that allows you to manage and provision infrastructure using code. It is commonly used for infrastructure as code (IaC) and is known for its ability to work across multiple cloud providers, reusability, versioning, flexibility, and its active community. Terraform providers are plugins that enable Terraform to communicate and manage resources on various cloud platforms such as AWS, Google Cloud, and others.
```
Comment (adela-bytebase) [inline content/blog/introducing-terraform-bytebase-provider.md:13]:
```suggestion
[Terraform](https://www.hashicorp.com/products/terraform) is an open-source tool that allows you to manage and provision infrastructure using code. It is commonly used for infrastructure as code (IaC) and is known for its ability to work across multiple cloud providers, reusability, versioning, flexibility, and its active community. Terraform providers are plugins that enable Terraform to communicate and manage resources on various cloud platforms such as AWS, Google Cloud, and others.
```

**Review by tianzhou — COMMENTED** [review]
Please edit per offline discussion

File: content/blog/introducing-terraform-bytebase-provider.md (hunk 1 above)
Comment (Candybase) [reply content/blog/introducing-terraform-bytebase-provider.md:15]: done

File: content/blog/introducing-terraform-bytebase-provider.md (hunk 2 above)
Comment (Candybase) [reply content/blog/introducing-terraform-bytebase-provider.md:16]: It should be removed

**Comment by Candybase** [comment]
Create a new branch "terraform-announcement" instead of "terrafrom-announcement".

**Pull request closed without merging**